client/blocks/app-promo/test/index.jsx

```diff
@@ -1,18 +1,13 @@
+/** @jest-environment jsdom */
```

Comment:
I can give it a try. Total execution time should be 30% longer according to this tests: https://medium.com/@vitaliypotapov/thanks-for-suggestion-jest-is-really-30-faster-with-env-node-option-the-chart-4309ba8aba81.